code/modules/clothing/rogueclothes/neck.dm

```diff
@@ -225,6 +221,20 @@
 		REMOVE_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT)
 		REMOVE_TRAIT(user, TRAIT_PACIFISM, CURSED_ITEM_TRAIT)
 
+
+/obj/item/clothing/neck/roguetown/gorget/servant/imprisoned
+	name = "enchanted prisoner collar"
+	desc = "This collar makes me obligated to heed to orders from anyone, and restricts my movements..."
+
+/obj/item/clothing/neck/roguetown/gorget/servant/imprisoned/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(slot == SLOT_NECK)
+		user.visible_message(span_warning("[user] is bound by the collar's enchantment."), \
+			span_warning("This collar makes me heed to orders from anyone, unless those orders will result in self harm or forced sexual interactions."))
```

Comment:
This directly contravenes the recent announcement stating that Public Servants are wards of the town of Stonehedge and only need to obey commands from town and guild officials and Yeoman.